### ISSUE TITLE
More tags for Replay Gain

### ISSUE THREAD

**Issue by RafaelFrance** [quodlibet/quodlibet#3254]
The proposed change
-------------------
To add tags in accordance of Replay Gain 2.0.
- Replaygain_track_peak
- Replaygain_track_range
- Replaygain_album_peak
- Replaygain_album_range
- Replaygain_reference_loudness. Example : -23 LUFS or -18 LUFS

To be able to search with these new tags.

Benefits of this change
-----------------------
All these tags help to a better understanding of audio properties of tracks.
There are some software who calculate and write these tags in tracks, but today Quod Libet can not show them. 

Any issues this might cause
---------------------------
The Replay Gain plugin should be updated to calculate Replay Gain 2.0 values, at least for Gain and Reference Loudness tags.

**Comment by declension** [comment]
We should check whether the gstreamer plugin has support for these values first...

**Comment by RafaelFrance** [comment]
Yes, but in any case it is possible to calculate and write values into audio files tags without Quod Libet. 
I do it today, but I can not use them with the so powerful search capabilities that I love so much and I use all time in Quod Libet.
If the plugin can not write these tags, I think it is not an problem.

**Comment by CreamyCookie** [comment]
See also #1609

**Comment by declension** [comment]
Relevant: https://github.com/metabrainz/picard/pull/1263
